pandas/tests/io/json/test_json_table_schema.py

```diff
@@ -334,57 +336,57 @@ def test_date_format_raises(self):
         self.df.to_json(orient='table', date_format='iso')
         self.df.to_json(orient='table')
 
-    def test_make_field_int(self):
+    def test_convert_pandas_type_to_json_field_int(self):
         data = [1, 2, 3]
         kinds = [pd.Series(data, name='name'), pd.Index(data, name='name')]
         for kind in kinds:
-            result = make_field(kind)
+            result = convert_pandas_type_to_json_field(kind)
             expected = {"name": "name", "type": 'integer'}
             assert result == expected
 
-    def test_make_field_float(self):
+    def test_convert_pandas_type_to_json_field_float(self):
         data = [1., 2., 3.]
         kinds = [pd.Series(data, name='name'), pd.Index(data, name='name')]
         for kind in kinds:
-            result = make_field(kind)
+            result = convert_pandas_type_to_json_field(kind)
             expected = {"name": "name", "type": 'number'}
             assert result == expected
 
-    def test_make_field_datetime(self):
+    def test_convert_pandas_type_to_json_field_datetime(self):
         data = [1., 2., 3.]
         kinds = [pd.Series(pd.to_datetime(data), name='values'),
                  pd.to_datetime(data)]
         for kind in kinds:
-            result = make_field(kind)
+            result = convert_pandas_type_to_json_field(kind)
             expected = {"name": "values", "type": 'datetime'}
             assert result == expected
 
         kinds = [pd.Series(pd.to_datetime(data, utc=True), name='values'),
                  pd.to_datetime(data, utc=True)]
         for kind in kinds:
```

Comment:
should come back later and parametrize these tests

Reply:
Opened #19070 to handle this